rebuild_all.sh

```diff
@@ -13,6 +13,7 @@ declare -a ALL_BOARDS=("com.matternet.carriage_2.0"
                        "com.matternet.ucann_1.0"
                        "com.matternet.bmu-B_1.0"
                        "com.matternet.external_led_1.0"
+                       "com.matternet.bmu_4.3"
```

Comment:
Is BMU now major.minor.patch? should this be 4.3.0?

Reply:
This is a hardware version.  Can we do minor patches on hardware?

Reply:
Ah, I missed that this was hw version. The danger of hw version == sw version :)